src/transmuxer_worker.js

```diff
@@ -88,15 +88,24 @@ var messageHandlers = {
     transmuxer.push(segment);
   },
   /**
-   * resetTransmuxer
+   * reset
    * Recreate the transmuxer so that the next segment added via `push`
-   * begins at a baseMediaDecodeTime of 0
+   * start with a fresh transmuxer
    */
-  resetTransmuxer: function (data) {
-    // delete the transmuxer
+  reset: function () {
     this.defaultInit();
   },
   /**
+   * setTimestampOffset
+   * Set the value that will be used as the `baseMediaDecodeTime` time for the
+   * next segment pushed in. Subsequent segments will have their `baseMediaDecodeTime`
+   * set relative to the first based on the PTS values.
+   */
+  setTimestampOffset: function (data) {
+    var timestampOffset = data.timestampOffset || 0;
+    transmuxer.setBaseMediaDecodeTime(Math.round(timestampOffset * 90000));
```

Comment:
The worker *is* taking seconds. The Transmuxer is taking 90k time. This function is the bridge between the two. I want to keep `videojs-media-sources.js` as transmuxer-agnostic as possible so that it can just worry about seconds where ever possible. Likewise, I'd like the time in the Transmuxer to know about the 90khz clock.